app/controllers/HomeController.scala

```diff
@@ -227,7 +234,7 @@ private def handleErrorGettingDueDates(isAgent: Boolean)(implicit user: MtdItUse
 }
 
   private def getCurrentITSAStatus(taxYear: TaxYear)(implicit user: MtdItUser[_]): Future[ITSAStatus.ITSAStatus] = {
-    ITSAStatusService.getStatusTillAvailableFutureYears(taxYear.previousYear).map(_.view.mapValues(_.status)
+    ITSAStatusService.getStatusTillAvailableFutureYears(taxYear.previousYear, user.nino).map(_.view.mapValues(_.status)
```

Comment:
I would suspect we need to save mandation status in session cookie for supporting agents as well?